app/uk/gov/hmrc/helptosavereminder/controllers/EmailCallbackController.scala

```diff
@@ -41,52 +41,57 @@ class EmailCallbackController @Inject()(
 
   def handleCallBack(callBackReference: String): Action[AnyContent] = Action.async { implicit request =>
     {
-      request.body.asJson.get
+      request.body.asJson
+        .getOrElse(Json.toJson(""))
```

Comment:
No, previous behaviour was that no body would cause None.get exception without meaning.  Now it will be treated as a callback with no map.  I'd suggest you throw at least an Exception with a message , or otherwise BAD_REQUEST

Reply:
Yes, can be re-written with getOrElse

Reply:
Done